packages/core/src/template-compiler.ts

```diff
@@ -41,40 +41,50 @@ interface GlimmerSyntax {
   cacheKey: string;
 }
 
+const CACHE = new Map();
```

Comment:
This cache could be something that stats the file, and uses the cache if the file change. That might be a nice addition... thoughts @ef4